### PR TITLE
Missing dependency for NamespaceClassLoader

### PR DESCRIPTION
**Pull request by richardhj**
Using composer’s autoloading will do it too but this solution is the uncomplicated one

### CODE REVIEW

**Comment by bytehead** [comment]
The `NamespaceClassLoader` is a dependency of Isotope anyway.

**Comment by richardhj** [comment]
Not anymore…
It seems they dropped the dependency since they are using the composer's autoloader per default (see Isotope's README)

**Comment by bytehead** [comment]
You're right. But then I would rather use the composer autoloading as well.
Get rid of dependencies ;-)

**Comment by richardhj** [comment]
I'm fan of it too. Someone could check the latest commit :-)

File: config/autoload.php [hunk 1]
```diff
@@ -12,7 +12,10 @@
 /**
  * Register namespace
  */
-NamespaceClassLoader::add('Rhyme', 'system/modules/isotope_feeds/library');
+if (class_exists('NamespaceClassLoader'))
+{
+    NamespaceClassLoader::add('Rhyme', 'system/modules/isotope_feeds/library');
```
Comment (bytehead) [inline config/autoload.php:17]: It's probably not needed when we have composer autoloading, isn't it?

Comment (richardhj) [inline config/autoload.php:17]: It's a fallback for those not using composer. They can install the NamespaceClassLoader and will be able to use this extension too.

Comment (bytehead) [inline config/autoload.php:17]: Good point.
Poor not composer users...

Comment (bytehead) [inline config/autoload.php:17]: Isotope is not usable without composer anyway 😉

**Pull request merged**